prov/verbs/src/fi_verbs.c

```diff
@@ -116,6 +116,11 @@ static struct fi_provider fi_ibv_prov = {
 		(flags & (FI_COMPLETION | FI_TRANSMIT_COMPLETE | FI_DELIVERY_COMPLETE)))
 #define VERBS_COMP_READ(ep) VERBS_COMP_READ_FLAGS(ep, ep->info->tx_attr->op_flags)
 
+#define VERBS_DBG(subsys, ...) FI_DBG(&fi_ibv_prov, FI_LOG_##subsys, __VA_ARGS__)
+#define VERBS_INFO(subsys, ...) FI_INFO(&fi_ibv_prov, FI_LOG_##subsys, __VA_ARGS__)
```

Comment:
My preference is to not split FI_LOG_XXX names like this.  This makes it impossible to search for FI_LOG_XXX in the code.